src/UI/Components/InlineEditable/EditableField.tsx

```diff
@@ -60,6 +60,7 @@ export const EditableField: React.FC<Props> = ({
   };
   const onEditClick = () => {
     setEditable(true);
+    setError(null);
```

Comment:
So if I click again on the input field, the error is gone? 🤔 Doesn't sound like a correct UX